lnd.go

```diff
@@ -201,11 +209,23 @@ func lndMain() error {
 	if err := rpcServer.Start(); err != nil {
 		return err
 	}
-	sCreds, err := credentials.NewServerTLSFromFile(cfg.TLSCertPath,
-		cfg.TLSKeyPath)
+	cert, err := tls.LoadX509KeyPair(cfg.TLSCertPath, cfg.TLSKeyPath)
 	if err != nil {
 		return err
 	}
+	tlsConf := &tls.Config{
```

Comment:
I've added a comment about the selection criteria for the cipher suites and reordered them from strongest to weakest primitives, but client preference order has a stronger effect on negotiation (and I'm not sure ordering matters at all on the server).